service/src/main/java/uk/gov/hmcts/reform/fpl/service/RespondentPolicyService.java

```diff
@@ -33,16 +34,25 @@
 
         Applicant firstApplicant = caseData.getAllApplicants().get(0).getValue();
 
-        for (int i = 0; i < caseData.getRespondents1().size(); i++) {
-            Element<Respondent> respondentElement = caseData.getRespondents1().get(i);
+        List<Element<Respondent>> respondents = caseData.getRespondents1();
+        int numOfRespondents = respondents.size();
 
-            NoticeOfChangeAnswers noticeOfChangeAnswer = noticeOfChangeRespondentConverter.generateForSubmission(
-                respondentElement, firstApplicant);
+        for (int i = 0; i < 10; i++) {
+            OrganisationPolicy organisationPolicy;
+            SolicitorRole solicitorRole = SolicitorRole.values()[i];
 
-            OrganisationPolicy organisationPolicy = respondentPolicyConverter.generateForSubmission(
-                respondentElement, SolicitorRole.values()[i]);
+            if (i < numOfRespondents) {
+                Element<Respondent> respondentElement = respondents.get(i);
+                organisationPolicy = respondentPolicyConverter.generateForSubmission(solicitorRole, respondentElement);
```

Comment:
maybe better to use one espondentPolicyConverter.generateForSubmission(solicitorRole, Optional.<respondentElement>) instead of having 2 methods. In this way you don't need 2 branches